docs/node-mixin/alerts/alerts.libsonnet

```diff
@@ -184,6 +184,20 @@
               description: '{{ $labels.instance }} interface {{ $labels.device }} has encountered {{ printf "%.0f" $value }} transmit errors in the last two minutes.',
             },
           },
+          {
+            alert: 'NodeClockSkewDetected',
+            expr: |||
+              abs(node_timex_offset_seconds) > 300
```

Comment:
In kube-prometheus we have this alert with 50ms.
`abs(node_timex_offset_seconds{job="node-exporter"})  > 0.05`.

I think it's fine to set this to 50ms too.